ocaml/xapi/xapi.ml

```diff
@@ -956,7 +956,9 @@ let server_init() =
           | Some self ->
             debug "Waiting forever for cluster_host to gain an IP address";
             let ip = Xapi_mgmt_iface.(wait_for_clustering_ip ~__context ~self) in
-            debug "Got clustering IP %s, attempting to re-plug any remaining unplugged PBDs" ip;
+            debug "Got clustering IP %s, resyncing cluster_host %s" ip (Ref.string_of self);
+            Xapi_cluster_host.resync_host ~__context ~host;
```

Comment:
Should not be needed, PBD plug already does this. I think you can drop this commit.